### PR TITLE
Add: Author Component

### PR DESCRIPTION
**Pull request by thebeatlesphan**
image

### CODE REVIEW

File: components/AboutAuthor/AboutAuthor.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+import kid from "../../public/kid.png";
+import styles from "./AboutAuthor.module.css";
+
+// mock data/api
+const fetchMockData = (slug) => ({
+  id: 1234,
+  authorAvatar: kid,
+  authorUsername: "trinh",
+  authorHandle: "@trinhandandy",
```
Comment (HamzahMansour) [inline components/AboutAuthor/AboutAuthor.js:11]: would either of these just be their blog name or are they something separate? in the db right now there's displayName which is a full name `Firstname Lastname` and blogName, which is more of a handle/blog slug `TestBlog`

File: components/AboutAuthor/AboutAuthor.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  authorDescription:
+    "When the tangerine moose first heard about the concept of 'orange is the new black,' he couldn't help but feel a little personally offended.",
+  authorLinks: ["x", "y", "z"],
```
Comment (HamzahMansour) [inline components/AboutAuthor/AboutAuthor.js:14]: would these be social links? and we'd eventually display them as icons only?

**Review by HamzahMansour — DISMISSED** [review]
questions about the mock data but LGTM

File: components/AboutAuthor/AboutAuthor.js (hunk 2 above)
Comment (thebeatlesphan) [reply components/AboutAuthor/AboutAuthor.js:14]: yeah that was the idea. This was the inspiration.

image

File: components/AboutAuthor/AboutAuthor.js (hunk 1 above)
Comment (thebeatlesphan) [reply components/AboutAuthor/AboutAuthor.js:11]: Yeah i just merged all the recent APIs and am working on retrieving blogs now that I have an idea how you guys are working with the DB.
I can update this authorCard to fit our schema.

**Comment by thebeatlesphan** [comment]
Updated AuthorCard


image

**Review by zwwuu — DISMISSED** [review]

**Review by zwwuu — APPROVED** [review]

**Pull request merged**